Makefile

```diff
@@ -54,7 +54,7 @@ install: $(PYTHON_VENV)
 	$(VENV_BIN)pip$(PYTHON_BIN_POSTFIX) install --requirement $(REQUIREMENTS)
 	touch $@
 
-.venv/install-timestamp: .venv/timestamp setup.py $(REQUIREMENTS)
+.venv/install-timestamp: .venv/timestamp setup.py requirements.txt
```

Comment:
But we use $(REQUIREMENTS) everywhere in the makefile. Why do we use only requirements.txt here ?

Reply:
Because we install the version from requirements.txt :-)

Reply:
Oh, yes, sorry. Forget this comment :-)